pkg/maps/policymap/policymap.go

```diff
@@ -84,13 +113,37 @@ const SizeofPolicyKey = int(unsafe.Sizeof(PolicyKey{}))
 // +k8s:deepcopy-gen:interfaces=github.com/cilium/cilium/pkg/bpf.MapValue
 type PolicyEntry struct {
 	ProxyPort uint16 `align:"proxy_port"` // In network byte-order
-	Pad0      uint16 `align:"pad0"`
+	Flags     uint8  `align:"deny"`
+	Pad0      uint8  `align:"pad0"`
```

Comment:
>Have we been zeroing these pad fields on Cilium <= 1.8...?
>
>I'm not sure if we will actually remove the policymap during upgrades if the map attributes match the previous release. Could be an upgrade headache if we haven't been.

these fields have been "Pad0" since `1.0.4`

https://github.com/cilium/cilium/commit/fdd4bc1d76131aa270dd24c8dc9e30268f220975

Do you think it's still worth worry about this?